src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf16Utility.Validation.cs

```diff
@@ -433,5 +481,16 @@ static Utf16Utility()
             scalarCountAdjustment = tempScalarCountAdjustment;
             return pInputBuffer;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static uint Arm64MoveMask(Vector128<byte> value, Vector128<byte> initialMask, Vector128<byte> mostSignficantBitMask)
```

Comment:
Please refer to the conversation here: https://github.com/dotnet/runtime/pull/38653#discussion_r452942942

In principle the jit should optimize for readonly statics, so we could probably replace the `initialMask` and `mostSignficantBitMask` parameters with readonly static fields, to minimize leaking of implementation details.

Reply:
Thanks, I'll get it updated.